package.json

```diff
@@ -21,21 +29,23 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@figma/plugin-typings": "^1.62.0",
-    "@svgr/core": "^6.5.1",
+    "@figma/plugin-typings": "^1.83.0",
+    "@rollup/pluginutils": "^5.1.0",
     "@types/node": "^18.15.11",
     "@types/postcss-url": "^10.0.0",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
-    "@vitejs/plugin-react": "^3.1.0",
+    "@vitejs/plugin-react": "^4.2.1",
     "cross-os": "^1.5.0",
+    "npm-run-all": "^4.1.5",
     "postcss": "^8.4.21",
     "postcss-url": "^10.1.3",
+    "rimraf": "^5.0.5",
     "sass": "^1.60.0",
-    "typescript": "^4.9.3",
-    "vite": "^4.2.0",
-    "vite-plugin-generate-file": "^0.0.4",
-    "vite-plugin-singlefile": "^0.13.3",
-    "vite-plugin-svgr": "^2.4.0"
+    "typescript": "^5.3.0",
+    "vite": "^5.0.11",
+    "vite-plugin-generate-file": "^0.1.1",
+    "vite-plugin-singlefile": "^0.13.5",
+    "vite-svg-loader": "^5.1.0"
```

Comment:
`vite-svg-loader` is actually a Vue library. 

Its `*.svg?component` loader actually loads in a Vue Functional Component
![image](https://github.com/CoconutGoodie/figma-plugin-react-vite/assets/8591785/3b34e036-6917-4a3f-9f23-e696d4a5fc25)

I love this one, however I couldn't find a React version that has beautiful import queries 😢

Reply:
Okay, I converted to using [vite-plugin-svgr](https://www.npmjs.com/package/vite-plugin-svgr), and now it's loading correctly in Figma.

Reply:
I actually spent some time and crafted a React version of `vite-svg-loader` 😀 

https://github.com/iGoodie/vite-plugin-react-rich-svg

I'll accept your changes, because I freaking love them :D
Then probably migrate svg loading to `vite-plugin-react-rich-svg` 🎉 

Thanks once again for all the efforts! 💜